src/test/crypto/yahoo-finance-service.spec.ts

```diff
@@ -1,20 +1,37 @@
 import { Test, TestingModule } from '@nestjs/testing'
 import { YahooFinanceService } from '../../crypto/yahoo-finance.service'
+import * as dayjs from 'dayjs'
 
 describe('YahooFinanceServiceController', () => {
   let service: YahooFinanceService
 
-  beforeEach(async () => {
-    const module: TestingModule = await Test.createTestingModule({
-      providers: [
-        YahooFinanceService,
-      ],
-    }).compile()
-    service = await module.get(YahooFinanceService)
+  it('getHistoricalOfBTCWithoutParameterInFunctionWork', async () => {
+    const data = await new YahooFinanceService('BTC').getHistory()
+    expect(data.length > 0).toBeTruthy()
   })
 
-  it('getHistoricalOfBTCWithoutParametterInFunctionWork', async () => {
-    const data = await new YahooFinanceService('BTC').getHistory()
-    console.log(data)
+  it('getHistoricalOfBTCWithOnlyBeginDateParameterInFunctionWork', async () => {
+    const beginDate = dayjs('2022-02-22', 'YYYY-MM-JJ')
```

Comment:
add "let beginDate: Date" and add beforeEach to init beginDate = dayjs('2022-02-22', 'YYYY-MM-JJ')